sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3266,16 +3253,62 @@ SIREPO.app.directive('jobsList', function(requestSender, appState, $location, $s
                 );
             }
 
+            $scope.endSimulation = function(rowIndex) {
+                let s = $scope.data.rows[rowIndex][1];
```

Comment:
I think the backend should just return a dict with the data and we process it on the front-end. Initially the backend was written to send an array of data and the frontend would just display any array but that doesn't really work. It would clean up this code a lot.

Reply:
52ad443c4474fedc3a7e625ee53d4610033d3a46